DQM/Integration/python/clients/beamspotdip_dqm_sourceclient-live_cfg.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
+import FWCore.ParameterSet.Config as cms
+
+#
+process = cms.Process("BeamSpotDipServer")
+process.load("DQMServices.Core.DQM_cfg")
+
+# input
+# for live online DQM in P5
+process.load("DQM.Integration.config.inputsource_cfi")
+# for testing in lxplus
+# process.load("DQM.Integration.config.fileinputsource_cfi")
+
+# beamspot from database
+process.load("CondCore.CondDB.CondDB_cfi")
+
+process.BeamSpotDBSource = cms.ESSource("PoolDBESSource",
+  process.CondDB,
+  toGet = cms.VPSet(
+    cms.PSet(
+      record = cms.string('BeamSpotOnlineLegacyObjectsRcd'),
+      tag = cms.string("BeamSpotOnlineTestLegacy"),
+      refreshTime = cms.uint64(1)
+    ),
+    cms.PSet(
+      record = cms.string('BeamSpotOnlineHLTObjectsRcd'),
+      tag = cms.string("BeamSpotOnlineTestHLT"),
+      refreshTime = cms.uint64(1)
+    )
+  )
+)
```

Comment:
In #34348 the two beamspot tags (the production ones, not the "test" ones) were included in the GT.
So if you use a recent Global Tag (_i.e._ `> 113X_dataRun3_HLT_v3`) the two tags are already included and you should remove the ESSource, see for example commit https://github.com/cms-sw/cmssw/pull/34348/commits/61d0becd48590b36d36f518246e5cb0dc4dd3276 from the same PR.

Reply:
I forgot to mention you should still add the refreshTime customization so you should have something similar to:
https://github.com/cms-sw/cmssw/blob/master/DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py#L142-L159

Reply:
Done